indico/web/templates/forms/category_picker_widget.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'forms/base_widget.html' %}
+
+
+{% block html %}
+    <div class="i-form-field-fixed-width" data-tooltip-anchor>
+        <div id="category-title-{{ field.id }}" class="text-holder-box"></div>
+        <a class="i-button" id="category-picker-button-{{ field.id }}"
+           data-title='{% trans %}Choose category{% endtrans %}'>
+            {%- trans %}Choose category{% endtrans -%}
+        </a>
+        <input type="hidden" id="{{ field.id }}" name="{{ field.name }}">
```

Comment:
Not a blocker for the merge I'd say. We'll need it only for event creation from within a category management instead from the global dropdown. We can add it later.